python/tvm/runtime/module.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, handle):
         self.handle = handle
         self._entry = None
         self.entry_name = "__tvm_main__"
+        # TODO:(FrozenGene): support rpc
+        if self.type_key == 'GraphRuntimeFactory':
+            #from tvm.runtime.graph_runtime_factory import GraphRuntimeFactoryModule
+            self._entry = self.runtime_create
```

Comment:
To support multi models to select, we want to support `lib['default'](ctx)`. So I override `_entry` (used by `__call__`) and `__getitem__`. @tqchen Do you have more elegant ways to do it? Especially for `rpc`.

Reply:
@tqchen do you have any suggestion here? For `rpc`, I could think of one way is to use `ModuleGetTypeKey` to extract and check it is `GraphRuntimeFactory`. Then we go here. However, I think the way I do is a little hack (include the logic of `GraphRuntimeFactory`. Do you have any better idea? Thanks in advance.

Reply:
Perhaps we can just support the default function in the GraphRuntimeFactory? so we don't have to change Module

Reply:
As we are in rpc said, we want to select multi models in one shared library. If we just support `default`, it doesn't our purpose? If we don't want to change the module, maybe we could use auto type key wrapper in the rfc? That is to say when we call `tvm.runtime.load_module`, we will check the type key, if we are `GraphRuntimeFactory`, we go to `GraphRuntimeFactoryModule`  directly and we could do everything inside it.

Reply:
Oh, what i mean is that user can directly use `mod["default"](ctx)` to create the model, of course `mod["resnet18"](ctx)` still works

Reply:
Yes. This is easy to be done when we use `relay.build`. However, when we complete `export_library` and use `tvm.runtime.load_module` / `remote.load_module` (remote is rpc) is not easy. Because when we load it, its type is module. So, my previous reply means maybe we could use automatic type wrapper mentioned in original rfc , check the type key when we `load_module` and go to `GraphRuntimeFactoryModule` directly, then we don’t modify `module`. How about it?